x-pack/plugins/security/server/routes/telemetry/authentication_type.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createHash } from 'crypto';
+
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '..';
+import { wrapIntoCustomErrorResponse } from '../../errors';
+import { createLicensedRouteHandler } from '../licensed_route_handler';
+
+const COUNTER_TYPE = 'SecurityAuthType';
+
+export function defineTelemetryOnAuthTypeRoutes({
+  getAuthenticationService,
+  router,
+  usageCounter,
+}: RouteDefinitionParams) {
+  router.post(
+    {
+      path: '/internal/security/telemetry/auth_type',
+      validate: {
+        body: schema.nullable(
+          schema.object({
+            auth_type: schema.string(),
+            timestamp: schema.number(),
+            username_hash: schema.string(),
+          })
+        ),
+      },
+    },
+    createLicensedRouteHandler(async (context, request, response) => {
+      try {
+        let timestamp = new Date().getTime();
+        const {
+          auth_type: oldAuthType,
+          username_hash: oldUsernameHash,
+          timestamp: oldTimestamp,
+        } = request.body || {
+          auth_type: '',
+          username_hash: '',
+          timestamp,
+        };
+        const authUser = await getAuthenticationService().getCurrentUser(request);
+        const usernameHash = authUser
+          ? createHash('sha3-256').update(authUser?.username).digest('hex')
+          : '';
+
+        const elapsedTimeInHrs = (timestamp - oldTimestamp) / (1000 * 60 * 60);
+
+        if (
+          !authUser?.authentication_type ||
+          (authUser?.authentication_type && authUser?.authentication_type === '')
+        ) {
+          return response.badRequest({
+            body: { message: `Authentication type can not be empty` },
+          });
+        }
```

Comment:
I was able to trigger this 400 error when running this with Security disabled in Elasticsearch.

Question: is there any particular reason you wanted to do this? IMO it's not a bad request (the client can't change the request params to get a 200 result).

I ask because any 400 errors in the browser's network log will throw up red flags when troubleshooting other unrelated problems.

So I'm thinking that in this case, it might be better to just return a 204 empty response. WDYT?